component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -188,13 +188,21 @@ public static ProfileEntity buildEntityProfile(Profile profile, String restPath,
     userEntity.setAvatar(profile.getAvatarUrl());
     userEntity.setBanner(profile.getBannerUrl());
     try {
+      boolean isExternal = profile.getProperty(Profile.EXTERNAL) != null && (profile.getProperty(Profile.EXTERNAL)).equals("true");
```

Comment:
Do we already have external information in profile ? I dont remember that we put it

Reply:
yes , we have already stored the info 
https://github.com/Meeds-io/social/blob/786db1a3f5d59d7e4050c58067cd77b2678127df/component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java#L146

Reply:
:+1: